include/swift/AST/DiagnosticsSema.def

```diff
@@ -795,19 +795,19 @@ ERROR(unspaced_unary_operator,none,
       "unary operators must not be juxtaposed; parenthesize inner expression",
       ())
 
-ERROR(use_unresolved_identifier,none,
-      "use of unresolved %select{identifier|operator}1 %0", (DeclNameRef, bool))
-ERROR(use_unresolved_identifier_corrected,none,
-      "use of unresolved %select{identifier|operator}1 %0; did you mean '%2'?",
+ERROR(couldnt_find_defn,none,
+      "couldn't find definition %select{of|of operator}1 %0", (DeclNameRef, bool))
+ERROR(couldnt_find_defn_corrected,none,
+      "couldn't find definition %select{of|of operator}1 %0; did you mean '%2'?",
       (DeclNameRef, bool, StringRef))
 NOTE(confusable_character,none,
       "%select{identifier|operator}0 '%1' contains possibly confused characters; "
       "did you mean to use '%2'?",
       (bool, StringRef, StringRef))
-ERROR(use_undeclared_type,none,
-      "use of undeclared type %0", (DeclNameRef))
-ERROR(use_undeclared_type_did_you_mean,none,
-      "use of undeclared type %0; did you mean to use '%1'?", (DeclNameRef, StringRef))
+ERROR(couldnt_find_defn_of_type,none,
+      "couldn't find definition of type %0", (DeclNameRef))
+ERROR(couldnt_find_defn_of_type_did_you_mean,none,
+      "couldn't find definition of type %0; did you mean to use '%1'?", (DeclNameRef, StringRef))
```

Comment:
This is iffy of course, but I *do* think the full form (could not or cannot) would be more catchy and read better.

Reply:
I don't have a strong feeling about it either way. Since Holly has given a lot of thought to diagnostics and precise wording, I've left the decision up to her (noted in the PR text). 🙂

Reply:
Indeed, as a rule we don’t use contractions in diagnostic text, I thought that Diagnostics.md already mentioned that but apparently not...

Reply:
I agree with Varun that it doesn't make much of a difference, but I think we should use `cannot` for consistency.